lib/remove_person.rb

```diff
@@ -0,0 +1,91 @@
+#!usr/bin/ruby
+
+puts "Enter name of person to remove:"
+name = gets.strip!
+puts "\n###########################\n# Removing #{name}\n###########################"
+
+def short_name(name)
+  name.gsub(" ", "-").downcase
+end
+
+def first_name(name)
+  name.split(" ")[0]
+end
+
+##########################################
+# Remove person from people.yml          #
+##########################################
+
+people_yml_path = "#{Dir.pwd}/data/people.yml"
+tmp_file_path = "#{people_yml_path}.tmp"
+found_person = false
+
+File.open(tmp_file_path, "w") do |output|
+  File.open(people_yml_path, "r") do |input|
+    to_remove_block = false
+    input.each_line do |line|
+      if to_remove_block && line.strip.empty?
+        to_remove_block = false
+        next
+      end
+
+      next if to_remove_block
+
+      if line.include?("- name: \"#{name}\"")
+        puts "Found person: #{line}"
+        to_remove_block = true
+        found_person = true
+        next
+      end
+
+      output.write(line) unless to_remove_block
+    end
+  end
+end
+
+# Delete old file and write in new one
+File.delete(people_yml_path)
+File.rename(tmp_file_path, people_yml_path)
+
+puts "Could not find #{name} in 'people.yml'." unless found_person
+puts "Searching for people page links..."
+
+#############################################################
+# Search through posts and remove links to people/#{name}   #
+#############################################################
+
+people_page_link_regex = Regexp.new ".*\(\D*\/people##{short_name(name)}\).*"
+ag_people_page_link_regex = "'.*\(\D*\/people##{short_name(name)}\).*'"
+dead_link_count = 0
+
+source_files = `ag #{ag_people_page_link_regex} source/ -l`.split("\n")
```

Comment:
Good point. I will see if I can make an alternative version, perhaps using `grep` or a derivative of it (that doesn't require prior installation). Otherwise a readme note makes sense